src/components/KeyPairGeneration.js

```
@@ -125,13 +125,13 @@ const KeysGeneration = (props) => {
           {t("key_pair_question")}
         </Typography>
 
-        <Hidden xsDown>
+        <Box sx={{ display: { xs: 'none', sm: 'block' } }}>
           <a href="/about/#why-need-private-key" target="_blank">
```

Biome (2.4.6)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)